data/frontmatter.yaml

```diff
@@ -0,0 +1,18 @@
+# this file lets you add descriptions and keywords
+# to files without modifying the file.
+#
+# Useful for upstream pages where we don't want to
+# or can't add front matter to the source!
+#
+# To add descriptions and keywords for a page,
+# add a key with the path of the page file,
+# relative to the content directory. For upstream
+# page files, that's the target of the file mount.
+
+engine/reference/builder.md:
+  description: |
+    Find all the available commands you can use in a Dockerfile
+    and learn how to use them, including COPY, ARG, ENTRYPOINT, and more.
+  keywords: |
+    dockerfile, docker file, docker copy, dockerfile exec, docker entrypoint,
+    dockerfile entrypoint, dockerfile arg, docker args, entrypoint, shell dockerfile
```

Comment:
In follow-up we should at other remote pages with missing meta

Reply:
Yeah just added this as a start